packages/eslint-plugin-pf-codemods/src/rules/v6/dataCodemodsCleanup/dataCodemodsCleanupOutput.tsx

```diff
@@ -0,0 +1,13 @@
+import {
+  DualListSelector,
+  LoginMainFooterLinksItem,
+  MastheadLogo,
+} from "@patternfly/react-core";
+
+export const DataCodemodsCleanupInput = () => (
+  <>
+    <DualListSelector />
+    <LoginMainFooterLinksItem />
+    <MastheadLogo />
+  </>
+);
```

Comment:
Looks like the output file had the formatter ran on it after execution. Doesn't really matter now, but will if we ever implement auto checking of these tsx files in CI.

Reply:
Oops, I thought we should run prettier on these output files, I do it every time 🤦